cime_config/SystemTests/rxcropmaturity.py

```diff
@@ -255,19 +255,28 @@ def _run_make_fsurdat_all_crops_everywhere(self):
             raise RuntimeError(error_message)
 
         # Where we will save the fsurdat version for this test
-        self._fsurdat_out = os.path.join(self._path_gddgen, "fsurdat.nc")
+        path, ext = os.path.splitext(self._fsurdat_in)
+        dir_in, filename_in_noext = os.path.split(path)
+        self._fsurdat_out = os.path.join(self._path_gddgen, f"{filename_in_noext}.all_crops_everywhere{ext}")
```

Comment:
In this line Erik's PR adds _path_gddgen. I will let @ekluzek and @samsrabin decide if Erik's is the correct version.

Reply:
I think the only difference is that my version satisfies black/pylint.

Reply:
Oh good, you're right.